### PR TITLE
Added some comments to main.c

### PR DESCRIPTION
**Pull request by JairLanderos**
Some comments to main.c have been made.

### CODE REVIEW

**Comment by DevCode-C** [comment]
Arregla los conflicto de cambios, por favor.

File: App/Source/main.c
Comment (DevCode-C) [inline App/Source/main.c:1]: image

Comment (DevCode-C) [reply App/Source/main.c:1]: image
image

Comment (DevCode-C) [reply App/Source/main.c:1]: Resolved

**Review by DevCode-C — APPROVED** [review]
Resolved

**Comment by RicardoRuizD** [comment]
Good that you resolved the conflicts!

**Pull request merged**